editoast/src/views/documents.rs

```diff
@@ -48,17 +48,16 @@ pub enum DocumentErrors {
         (status = 404, description = "Document not found", body = InternalError),
     )
 )]
-#[get("/{document_key}")]
-async fn get(db_pool: Data<DbConnectionPoolV2>, document_key: Path<i64>) -> Result<HttpResponse> {
-    let document_key = document_key.into_inner();
+async fn get(
+    State(db_pool): State<DbConnectionPoolV2>,
+    Path(document_id): Path<i64>,
+) -> Result<impl IntoResponse> {
     let conn = &mut db_pool.get().await?;
-    let doc = Document::retrieve_or_fail(conn, document_key, || DocumentErrors::NotFound {
-        document_key,
+    let doc = Document::retrieve_or_fail(conn, document_id, || DocumentErrors::NotFound {
+        document_key: document_id,
```

Comment:
We usually have the suffix `_id` for variables holding a row PK. We didn't change the error variant as this would impact the public api of editoast. (We made sure that we wouldn't break the API bu watching changes made to `openapi.yaml`. Changing that would introduce "noise" in the diff.)

I agree that the error needs to change though. I'd favor a follow-up PR.

Reply:
It seems that we have a bunch of plans for follow-up PR, it'd be nice to make a list in the current PR of all the follow-up PR we expect, what do you think?

Reply:
I opened a tracking issue here: https://github.com/OpenRailAssociation/osrd/issues/8178